.env.defaults

```diff
@@ -11,6 +11,8 @@ NODE_JWK_KEY=''
 PUBSUB_TYPE=redis
 # required only if PUBSUB_TYPE=streamr
 STREAMR_STREAM_ID=''
+STREAMR_STREAM_HOST=''
+STREAMR_STREAM_PORT=''
```

Comment:
btw. - since we now have w subscribe-only account for the Redis - can you please add Redis configuration to `.env.defaults`? (in the `Warp GW Redis connection` section)